.gitignore

```diff
@@ -28,3 +28,6 @@ hs_err_pid*
 
 # macOS files
 .DS_Store
+
+# vim
+*.swp
```

Comment:
Not sure if it’s worth doing for three lines, but perhaps this could be split into another PR?

Reply:
bruh

Reply:
Gotta be kidding me. It is separated in the commit history.

Reply:
> Not sure if it’s worth doing for three lines, but perhaps this could be split into another PR?

What three lines? Have you looked at the changes?

Reply:
> bruh. It is separated in the commit history.

the commit has to be in a different branchfor it to not get added to the existing PR

Reply:
> > bruh. It is separated in the commit history.
> 
> the commit has to be in a different branch for it to not get added to the existing PR

I know.

Reply:
> > > bruh. It is separated in the commit history.
> > 
> > 
> > the commit has to be in a different branch for it to not get added to the existing PR
> 
> I know.

It isn't though: https://github.com/hmelder/pgdp2223-tests/commit/18273adb6f53723196e05963d3be243dc26ebd2c

that's why it is linked to this PR.

Reply:
> > > > bruh. It is separated in the commit history.
> > > 
> > > 
> > > the commit has to be in a different branch for it to not get added to the existing PR
> > 
> > 
> > I know.
> 
> It isn't though: [hmelder@18273ad](https://github.com/hmelder/pgdp2223-tests/commit/18273adb6f53723196e05963d3be243dc26ebd2c)
> 
> that's why it is linked to this PR.

I never contemplated that. This commit was intentionally added to this branch.

Reply:
It is not in the same commit as the w09h02 changes. You could squash it differently if you want to, but what do you gain?

Reply:
I have created a separate PR #197 to follow the pattern of small and separated PRs 👍